README.md

```diff
@@ -138,6 +138,7 @@ const Page = () => (
     <NextSeo
       title="Using More of Config"
       description="This example uses more of the available config options."
+      colorTheme="#73fa97"
```

Comment:
```suggestion
```

removing this as it should be `themeColor`

I will add it to the docs rather than suggesting it as a default.